packages/Popover/README.md

```diff
@@ -0,0 +1,106 @@
+# Popover
+
+## Example
+
+```js
+<button
+  className={containerStyle}
+  onClick={() => this.setState({ active: !this.state.active })}
+>
+  Popover
+  <Popover
+    align={select('Align', ['top', 'bottom', 'left', 'right'], 'bottom')}
+    justify={select('justify', ['start', 'middle', 'end'], 'start')}
+    active={this.state.active}
+    usePortal={boolean('usePortal', true)}
+    spacing={number('spacing', 10)}
+  >
+    <div className={popoverStyle}>Popover content</div>
+  </Popover>
+</button>
+```
+
+## Output HTML
+
+```html
+<button class="leafygreen-ui-79elbk">
+  Popover
+  <div class="leafygreen-ui-1hyfx7x"></div>
+</button>
+
+<div align="bottom" justify="start" class="leafygreen-ui-1t5dnko">
+  <div class="leafygreen-ui-ii2v5b">Popover content</div>
+</div>
+```
+
+## Simple Use Case
+
+The popover component will be automatically positioned relative to its neartest parent, which has position `absolute` specified.
```

Comment:
Typo - "neartest".

Also, not entirely true. It will default to being positioned relative to its direct parent, regardless of its positioning. The only time it matters what the parent's position is, is when `usePortal: false`. In that case, it will be relative to its nearest descendant that has `position: absolute | relative | fixed`.

Reply:
nearest ancestor, not descendant :)

Reply:
Fixed 👍

Reply:
@jetpacmonkey Yep! My bad